library/Ivoz/Provider/Domain/Service/Administrator/AdminLoginChecker.php

```diff
@@ -2,13 +2,20 @@
 
 namespace Ivoz\Provider\Domain\Service\Administrator;
 
+use Symfony\Component\HttpFoundation\RequestStack;
```

Comment:
Request is an infraestruture concept, it should not be part of a domain service

Reply:
I have moved it to a service within the infrastructure, injected via a domain interface. We didn't have anything like this before, so I'm uncertain about how to name the components and organize the folders.

I'm really looking forward to hearing your opinion about that.